bin/jade.js

```diff
@@ -58,6 +59,17 @@ program.on('--help', function(){
 
 program.parse(process.argv);
 
+// locals given, parse them
+
+var locals;
+if (program.locals) {
+  if (exists(program.locals)) {
+    locals = JSON.parse(fs.readFileSync(program.locals));
```

Comment:
OK! It is not to bother ... I do it in the best of the spirits! is that your work is hard!